docs/sources/features/panels/singlestat.md

```diff
@@ -23,17 +22,17 @@ The singlestat panel has a normal query editor to allow you define your exact me
 {{< docs-imagebox img="/img/docs/v45/singlestat-value-options.png" class="docs-image--no-shadow" max-width="900px" >}}
 
 1. **Stats**: The Stats field let you set the function (min, max, average, current, total, first, delta, range) that your entire query is reduced into a single value with. This reduces the entire query into a single summary value that is displayed.
```

Comment:
If you'd like to improve further, I'd recommend using lazy list ordering, especially for longer lists, i.e.:

```
1. First item
1. Second item
1. Third item
```